homeassistant/components/zha/switch.py

```diff
@@ -588,3 +589,62 @@ class AqaraBuzzerManualAlarm(ZHASwitchConfigurationEntity):
     _attribute_name = "buzzer_manual_alarm"
     _attr_translation_key = "buzzer_manual_alarm"
     _attr_icon: str = "mdi:bullhorn"
+
+
+@CONFIG_DIAGNOSTIC_MATCH(cluster_handler_names="window_covering")
```

Comment:
(for another PR: we use constants for cluster handlers like `CLUSTER_HANDLER_ON_OFF`, but not for this. Could be cleaned up slightly)